api/chat_main_handler.go

```diff
@@ -71,12 +70,11 @@ type Message struct {
 }
 
 type OpenaiChatRequest struct {
-	Model    string    `json:"model"`
+	Model    string                         `json:"model"`
 	Messages []openai.ChatCompletionMessage `json:"messages"`
-
 }
 
-func NewUserMessage(content string) openai.ChatCompletionMessage{
+func NewUserMessage(content string) openai.ChatCompletionMessage {
 	return openai.ChatCompletionMessage{Role: "user", Content: content}
 }
 
```

Comment:
:robot: OpenAI

This patch only changes the formatting of the `NewUserMessage` function.

The function's functionality remains unchanged, so this change does not introduce any new functionality or bug risks.

Please confirm that the formatting change is intentional before merging this patch.

Also, please update the comments on the pull request to reflect this change.

<!-- This is an auto-generated comment by OpenAI -->